base/test/BUILD.gn

```diff
@@ -317,6 +317,12 @@ static_library("test_support") {
     ]
   }
 
+  if (!use_blink && use_cobalt_customizations) {
+    sources += [
+      "multiprocess_test.cc",
```

Comment:
I don't think we want to run this test, is it being added for some other reason?